src/components/SubmissionForm/ProjectDetails.js

```diff
@@ -98,66 +129,73 @@ const ProjectDetails = props => {
               </MenuItem>
             ))}
         </TextField>
-        <br />
         <TextField
           className="submitInput"
+          variant="outlined"
           type="text"
           value={name} /*???*/
           required
           name="name"
           id="standard-required"
-          placeholder="App Name*"
-          margin="normal"
+          label="App Name"
+          // margin="normal"
```

Comment:
Is this commented out `margin="normal"` doing anything?

Reply:
Wasn't doing anything with fixes I made. Removed.